diff --git a/dist/client/index.js b/dist/client/index.js
index 75b46470f2b74908cacb1812b105239d52678f9c..2e75a772f0e3913a13702f61339e47b7dd0cdf9c 100644
--- a/dist/client/index.js
+++ b/dist/client/index.js
@@ -6,7 +6,7 @@ const RUNTIME_CLIENT_RUNTIME_PATH = "/@vite-plugin-checker-runtime";
 const RUNTIME_CLIENT_ENTRY_PATH = "/@vite-plugin-checker-runtime-entry";
 const wrapVirtualPrefix = (id) => `virtual:${id.slice(1)}`;
 const composePreambleCode = ({ baseWithOrigin = "/", overlayConfig }) => `
-import { inject } from "${baseWithOrigin}${RUNTIME_CLIENT_RUNTIME_PATH.slice(1)}";
+import { inject } from "${RUNTIME_CLIENT_RUNTIME_PATH}";
 inject({
   overlayConfig: ${JSON.stringify(overlayConfig)},
   base: "${baseWithOrigin}",
